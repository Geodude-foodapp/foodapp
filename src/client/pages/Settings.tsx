import React from "react";
import { render } from "react-dom";
import { UserData } from '../../Types';


type SettingsProps = {
  setUserData: (data: UserData) => void;
};

//TODO: update with info from user data object under diet and preferences
export default ({setUserData }: SettingsProps) => (
  <section id="settings">
    <div className="settings-container">
      <h1>Settings</h1>
      <div className="current-preferences">
        <h3>Current Preferences:</h3>
        <h4>Diet</h4>
        {}
        <h4>Intolerances</h4>
        <button>Update Preferences</button>
      </div>
      <button>Change Password</button>
    </div>
  </section>
);
