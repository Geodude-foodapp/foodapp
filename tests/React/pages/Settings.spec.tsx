import { render } from '@testing-library/react';
import React from 'react';
import Settings from '../../../src/client/pages/Settings';

describe('Settings Page', () => {
  beforeAll(() => render(<Settings />));
  it('matches the snapshot', () => {
    const settingsPage = document.querySelector('#settings');
    expect(settingsPage).toMatchSnapshot();
  });
});
