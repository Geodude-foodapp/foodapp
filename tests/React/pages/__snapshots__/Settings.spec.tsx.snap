// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings Page matches the snapshot 1`] = `
<section
  id="settings"
>
  <h1>
    Settings
  </h1>
  ;
</section>
`;
